app/ante/evm/01_setup_ctx.go

```diff
@@ -24,26 +24,24 @@ func NewEthSetUpContextDecorator(evmKeeper EVMKeeper) EthSetupContextDecorator {
 	}
 }
 
-func (esc EthSetupContextDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simulate bool, next sdk.AnteHandler) (newCtx sdk.Context, err error) {
+func (esc EthSetupContextDecorator) AnteHandle(ctx sdktypes.Context, tx sdktypes.Tx, simulate bool, next sdktypes.AnteHandler) (newCtx sdktypes.Context, err error) {
 	newCtx, err = SetupContext(ctx, tx, esc.evmKeeper)
 	if err != nil {
 		return ctx, err
 	}
 	return next(newCtx, tx, simulate)
 }
 
-func SetupContext(ctx sdk.Context, tx sdk.Tx, evmKeeper EVMKeeper) (sdk.Context, error) {
+func SetupContext(ctx sdktypes.Context, tx sdktypes.Tx, evmKeeper EVMKeeper) (sdktypes.Context, error) {
```

Comment:
not sure why this was renamed? 🤔 I think we are calling it `sdk.Context` etc in most of the repository